### PR TITLE
quiz endpoint

### PR DESCRIPTION
**Pull request by loievskyi**

### CODE REVIEW

File: src/sections/quiz/quiz-header.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import {Lora} from '@next/font/google';
+
+const lora = Lora({preload: false});
+
+export default function QuizHeader() {
+  return (
+    <Box
+      component="header"
+      sx={{
+        backgroundColor: '#f9f4ee',
```
Comment (shuryhin-oleksandr) [inline src/sections/quiz/quiz-header.js:15]: Customize palette and refer to value from palette here.

File: src/sections/quiz/quiz-header.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        padding: '24px',
```
Comment (shuryhin-oleksandr) [inline src/sections/quiz/quiz-header.js:16]: Use responsive values, i.e. padding: {3}. That is a value relative to a current font size. This way spacing will scale together with font changes, set in theme typography.

File: src/sections/quiz/quiz-header.js (hunk 1 above)
Comment (loievskyi) [reply src/sections/quiz/quiz-header.js:15]: backgroundColor: theme.palette.background.quiz,

File: src/sections/quiz/quiz-header.js (hunk 2 above)
Comment (loievskyi) [reply src/sections/quiz/quiz-header.js:16]: I have a syntax error when padding: {3},
changed to 1.8em
image

File: src/sections/quiz/quiz-view.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+'use client';
+
+
+import QuizHeader from "./quiz-header";
+
+export default function QuizView() {
+  return (
+    <>
+      {<QuizHeader/>}
```
Comment (shuryhin-oleksandr) [inline src/sections/quiz/quiz-view.js:9]: No need for {} around QuizHeader

File: src/sections/quiz/quiz-header.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+const lora = Lora({preload: false});
+
+export default function QuizHeader() {
+  const theme = useTheme()
+  return (
+    <Box
+      component="header"
+      sx={{
+        backgroundColor: theme.palette.background.quiz,
+        padding: '1.7em',
```
Comment (shuryhin-oleksandr) [inline src/sections/quiz/quiz-header.js:19]: Could be just 1.7 instead of '1.7em'

File: src/sections/quiz/quiz-header.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        alignItems: 'center',
```
Comment (shuryhin-oleksandr) [inline src/sections/quiz/quiz-header.js:20]: We have a single element here, no need to align it.

File: src/sections/quiz/quiz-header.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+      }}
+    >
+      <Container maxWidth="lg" sx={{
+        display: "flex",
+        justifyContent: "space-between",
+      }}>
+        <SvgIcon component={Logo} inheritViewBox sx={{
```
Comment (shuryhin-oleksandr) [inline src/sections/quiz/quiz-header.js:27]: It is empty, could be written as <SvgIcon ... />

File: src/sections/quiz/quiz-header.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+          width: 321,
+          height: 64,
+          top: 28,
```
Comment (shuryhin-oleksandr) [inline src/sections/quiz/quiz-header.js:30]: Element has a static position, top and left do nothing.

File: src/sections/quiz/quiz-header.js [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+      component="header"
+      sx={{
+        backgroundColor: theme.palette.background.quiz,
+        padding: '1.7em',
+        alignItems: 'center',
+      }}
+    >
+      <Container maxWidth="lg" sx={{
+        display: "flex",
+        justifyContent: "space-between",
```
Comment (shuryhin-oleksandr) [inline src/sections/quiz/quiz-header.js:25]: The same staff, we have a single element inside, no need to align it with flex and justifyContent

File: src/sections/quiz/quiz-header.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
+import Button from "@mui/material/Button";
```
Comment (shuryhin-oleksandr) [inline src/sections/quiz/quiz-header.js:1]: Unused import

File: src/sections/quiz/quiz-header.js [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+import {SvgIcon} from "@mui/material";
+import Container from "@mui/material/Container";
+import Box from "@mui/material/Box";
+import Logo from "../../components/svg-color/Logo.svg";
+import {Lora} from '@next/font/google';
+import { useTheme } from '@mui/material/styles';
+
+
+const lora = Lora({preload: false});
```
Comment (shuryhin-oleksandr) [inline src/sections/quiz/quiz-header.js:10]: Unused variable

**Pull request merged**